src/functions/productById/handler.ts

```
@@ -0,0 +1,14 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import products from '@constants/products.json';
+import schema from './schema';
+
+const productList: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+  const { productId } = event.pathParameters;
+  const result = products.filter(prod => prod.id === productId);
```

Product not found error is not handled.